src/api/index.ts

```diff
@@ -0,0 +1,7 @@
+import { IArtist } from "types";
```

Comment:
I know this is a little boring, but if we deploy on vercel api folder messes up with nextjs, then can we name the folder `apis` over `api`?